prime-router/src/main/resources/metadata/hl7_mapping/receivers/STLTs/MD/MD-receiver-transform.yml

```diff
@@ -0,0 +1,38 @@
+hl7Class: ca.uhn.hl7v2.model.v251.message.ORU_R01
+
+extends: classpath:/metadata/hl7_mapping/ORU_R01/ORU_R01-base.yml
+constants:
+  PID: '/PATIENT_RESULT/PATIENT/PID'
+
+elements:
+
+  - name: xtn-use-code-from-system
+    condition: '%resource.system = "email"'
+    value: [ '"NET"' ]
+    hl7Spec: [ '%{hl7TelecomPath}-2' ]
+
+  - name: md-message-encoding-characters
+    condition: true
+    value: [ '"^~\\&"' ]
+    hl7Spec: [ 'MSH-2' ]
+
+  - name: md-sending-application-namespace-id
+    condition: true
+    value: [ '"CDC PRIME - Atlanta, Georgia (Dekalb)"' ]
+    hl7Spec: [ 'MSH-3-1' ]
+
+  - name: md-OID-of-the-sending-application-universal-id
+    condition: true
+    value: [ '"2.16.840.1.114222.4.1.237821"' ]
+    hl7Spec: [ 'MSH-3-2' ]
+
+  - name: md-sending-application-universal-id-type
+    condition: true
+    value: [ '"ISO"' ]
+    hl7Spec: [ 'MSH-3-3' ]
+
+  - name: md-patient-race-coding-system
+    resource: 'Bundle.entry.resource.ofType(Patient).extension("http://ibm.com/fhir/cdm/StructureDefinition/local-race-cd").value.coding'
+    condition: '%resource.code.exists()'
+    value: [ '%resource.system.getCodingSystemMapping()' ]
+    hl7Spec: [ '%{PID}-10-3' ]
```

Comment:
Most STLTs use these transforms, we should look into turning into a re-usable transform

Reply:
@victor-chaparro Should we move it to sender transform (./src/main/resource/metadata/fhir_transforms/senders/SimpleReport/simple-report-sender-transform.yml)?